os/include/tinyara/usb/usb.h

```diff
@@ -264,6 +264,22 @@
 #define USBOTG_FEATURE_A_HNP_SUPPORT            4	/* A device supports HNP */
 #define USBOTG_FEATURE_A_ALT_HNP_SUPPORT        5	/* Another port on the A device supports HNP */
 
+#define USB_SPEED_UNKNOWN                       0	/* Transfer rate not yet set */
+#define USB_SPEED_LOW                           1	/* USB 1.1 */
```

Comment:
USB low speed version was 1.0 which has transfer rate of 1.5 Mbits/s
and then comes USB full speed 1.1 has transfer rate of 12Mbit/s

please do 1.1 --> 1.0

Reply:
Done